lib/routes.dart

```diff
@@ -1,11 +1,14 @@
 import 'package:flutter/material.dart';
 import 'package:mskirana_app/ui/home/home_page.dart';
 import 'package:mskirana_app/ui/login/login_page.dart';
+import 'package:mskirana_app/ui/order/order_page.dart';
 import 'package:mskirana_app/ui/tracking/track_page.dart';
 
 final routes = {
   '/account': (BuildContext context) => new HomePage(),
   '/tracking': (BuildContext context) => new TrackPage(),
   '/home': (BuildContext context) => new HomePage(),
-  '/': (BuildContext context) => new LoginPage()
+  '/': (BuildContext context) => new LoginPage(),
+  '/order': (BuildContext context) => new OrderPage(),
```

Comment:
Hey, for consistency, can we add order above '/home' and below '/tracking'?

Reply:
Latest code should have it in the required order